fdbserver/DataDistribution.actor.cpp

```diff
@@ -1321,6 +1324,22 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 		addTeam(newTeamServers, isInitialTeam);
 	}
 
+	ACTOR static Future<Void> notifySSTeamChange(Optional<Reference<AsyncVar<struct ServerDBInfo>>> dbInfo)
+	{
+		if (dbInfo.present()) {
+			state Future<ErrorOr<Void>> proxyReply = dbInfo.get()->get().client.commitProxies[0].ssTeamChange.tryGetReply(SSTeamChangeRequest());
```

Comment:
This doesn't necessarily need to be fixed to proxy 0. You can do things similar as https://github.com/jzhou77/foundationdb/blob/c8f90693edaeace2ce04356bd42d981a57ef13a3/fdbserver/storageserver.actor.cpp#L5240-L5248
by using `basicLoadBalance`.